client/src/components/SessionManagerField.js

```diff
@@ -1,20 +1,15 @@
 /* global window */
-import React, { Component } from 'react';
+import React from 'react';
+import LoginSession from './LoginSession';
 
-class SessionManagerField extends Component {
-  render() {
+function SessionManagerField(props) {
     return (
-      <div>
-        <p>hello world</p>
+      <div className={'SessionManagerField'}>
```

Comment:
```suggestion
      <div className="session-manager-field">
```
Naming convention for classes in admin module is to have lowercase with dashes between words [example](https://github.com/silverstripe/silverstripe-admin/blob/1/client/src/components/Button/BackButton.js#L8)